render/gbm_allocator.c

```diff
@@ -159,7 +159,11 @@ static struct wlr_gbm_allocator *get_gbm_alloc_from_alloc(
 	return (struct wlr_gbm_allocator *)alloc;
 }
 
-struct wlr_gbm_allocator *wlr_gbm_allocator_create(int fd) {
+struct wlr_allocator *wlr_gbm_allocator_create(int fd) {
+	char *drm_name = drmGetDeviceNameFromFd2(fd);
+	wlr_log(WLR_DEBUG, "Using DRM node %s", drm_name);
```

Comment:
Maybe we can log this on the same line as "Created GBM allocator …" at the end of the function?